### ISSUE TITLE
[Fixed in GH-6052] Safari: Missing Visual Focus on My Account: Change Password

### ISSUE THREAD

**Issue by developpeurweb** [SAP/cloud-commerce-spartacus-storefront#4334]
All input fields and buttons missing visual focus

![Screen Shot 2019-08-29 at 9 00 22 PM](https://user-images.githubusercontent.com/15113219/63986044-2ff9f800-caa0-11e9-8620-7878432ef1c8.png)

**Comment by kacperknapik** [comment]
- Chrome: ✔
- Firefox: ✔ (note: focus border is huge)
- Safari: ❓

**Comment by KateChuen** [comment]
Chrome & Firefox: OK

Missing the keyboard accessibility and visual focus on:
- Cancel & Save buttons on Safari

**Comment by KateChuen** [comment]
Tested on dev3. Missing visual focus on safari for:

- all the input fields
- all the buttons

**Comment by developpeurweb** [comment]
**For QA:**

- To test in Safari please refer to [these instructions](https://github.com/SAP/cloud-commerce-spartacus-storefront/issues/5041#issuecomment-558883497).
- This got fixed in ticket GH-6052
- Theme `calydon` should be enabled to see changes.
- Visible focus outline overlapping adjacent ui elements may happen and may not be an issue, if in doubt please open a ticket per element or reach out to UI developer.
- Please separate visible focus issues (CSS: element without outline) from tabbing issues (HTML/TS: element not reachable with keyboard).

![Screen Shot 2020-01-31 at 1 24 22 PM](https://user-images.githubusercontent.com/15113219/73564193-0bc68c80-442d-11ea-9521-b6963332b53e.png)
![Screen Shot 2020-01-31 at 1 24 34 PM](https://user-images.githubusercontent.com/15113219/73564194-0bc68c80-442d-11ea-91c1-4fc53c776ecf.png)

**Comment by KateChuen** [comment]
Tested on dev3.
- Chrome ✅
- Firefox ✅
- Safari ✅
